etc/config.sample.toml

```diff
@@ -97,6 +97,15 @@
   # to cache snapshotting.
   # max-concurrent-compactions = 0
 
+  # CompactThroughput is the rate limit in bytes per second that we
+  # will allow TSM compactions to happen. Note that short bursts are allowed
```

Comment:
The feature is to prevent long IO blocking, so instead of "compaction to happen." how about "compactions to write to disk.".